testsuite/tests/typing-modules-bugs/pr9695_bad.ml

```diff
@@ -0,0 +1,10 @@
+(* TEST
+flags = " -w a -no-alias-deps"
+ocamlc_byte_exit_status = "2"
+* setup-ocamlc.byte-build-env
+** ocamlc.byte
+*** check-ocamlc.byte-output
+*)
```

Comment:
Is there a reason for not using an `* expect` tests with `flags="-no-alias-deps"`?

Reply:
I'm not sure that it would work (it seems the top-level errored on the alias rather than on the open)